### PR TITLE
Documented two additional typedefs

### PR DESCRIPTION
**Pull request by rpetrusha**
## Documented two additional typedefs

//cc @davmason @noahfalk @mjsabby

### CODE REVIEW

**Comment by davmason** [comment]
Looks good to me, thanks @rpetrusha

**Review by davmason — APPROVED** [review]

File: docs/framework/unmanaged-api/common-data-types-unmanaged-api-reference.md [hunk 1]
```diff
@@ -28,8 +28,10 @@ This topic lists simple data types used by the unmanaged APIs for the .NET Frame
 |mdToken|`typedef UINT32 mdToken;`|corprof.h|A metadata token (a row in a metadata table).|  
 |ModuleID|`typedef UINT_PTR ModuleID;`|corprof.h|The identifier of an assembly module.|  
 |ObjectID|`typedef UINT_PTR ObjectID;`|corprof.h|The identifier of an object.|  
+|PCCOR_SIGNATURE|`typedef SIZE_T PCCOR_SIGNATURE;`|cordebug.h|A pointer to a member or metadata signature.|
 |ProcessID|`typedef UINT_PTR ProcessID;`|corprof.h|The identifier of a managed process.|  
 |ReJITID|`typedef UINT_PTR ReJITID;`|corprof.h|The identifier of a jitted function.|  
+|SIZE_T|`typedef ULONG_PTR SIZE_T;`|corsym.h|A pointer to 64-bit memory address.|
```
Comment (mairaw) [inline docs/framework/unmanaged-api/common-data-types-unmanaged-api-reference.md]:
```suggestion
|SIZE_T|`typedef ULONG_PTR SIZE_T;`|corsym.h|A pointer to a 64-bit memory address.|
```

**Review by mairaw — APPROVED** [review]
LGTM. I left a small nit comment.

File: docs/framework/unmanaged-api/common-data-types-unmanaged-api-reference.md (hunk 1 above)
Comment (rpetrusha) [reply docs/framework/unmanaged-api/common-data-types-unmanaged-api-reference.md]: Those pesky articles, @mairaw!

**Review by mjsabby — APPROVED** [review]

**Pull request merged**